glean-core/src/lib.rs

```diff
@@ -874,6 +874,22 @@ pub(crate) fn register_ping_type(ping: &PingType) {
     }
 }
 
+/// Gets a list of currently registered ping names.
+///
+/// # Returns
+///
+/// The list of ping names that are currently registered.
+pub fn glean_get_registered_ping_names() -> Vec<String> {
+    block_on_dispatcher();
+    core::with_glean(|glean| {
+        let mut pings: Vec<String> = Vec::new();
+        glean.get_registered_ping_names().iter().for_each(|ping| {
+            pings.push(ping.to_string());
+        });
+        pings
```

Comment:
fwiw this could be changed to

```suggestion
        glean.get_registered_ping_names().iter().map(|ping| {
            ping.to_string()
        }).collect()
```

(I think)
But fine either way.